include/spdlog/details/spdlog_impl.h

```diff
@@ -31,6 +31,7 @@
 #include "../sinks/file_sinks.h"
 #include "../sinks/stdout_sinks.h"
 #include "../sinks/syslog_sink.h"
+#include "../sinks/android_sink.h"
```

Comment:
I was thinking that the only file user need to include to work with library is `#include "spdlog/spdlog.h"` so I guess we need to include `sinks/android_sink.h` somewhere internally. I've just found other sinks and add android one there.